frontend/catalyst/qjit_device.py

```diff
@@ -33,62 +32,83 @@
 )
 
 RUNTIME_OPERATIONS = {
+    "CNOT",
+    "ControlledPhaseShift",
+    "CRot",
+    "CRX",
+    "CRY",
+    "CRZ",
+    "CSWAP",
+    "CY",
+    "CZ",
+    "Hadamard",
     "Identity",
+    "IsingXX",
+    "IsingXY",
+    "IsingYY",
+    "ISWAP",
+    "MultiRZ",
     "PauliX",
     "PauliY",
     "PauliZ",
-    "Hadamard",
-    "S",
-    "T",
     "PhaseShift",
+    "PSWAP",
+    "QubitUnitary",
+    "Rot",
     "RX",
     "RY",
     "RZ",
-    "Rot",
-    "CNOT",
-    "CY",
-    "CZ",
+    "S",
     "SWAP",
-    "IsingXX",
-    "IsingYY",
-    "IsingXY",
-    "ControlledPhaseShift",
-    "CRX",
-    "CRY",
-    "CRZ",
-    "CRot",
-    "CSWAP",
+    "T",
     "Toffoli",
-    "MultiRZ",
-    "QubitUnitary",
-    "ISWAP",
-    "PSWAP",
     "GlobalPhase",
+    "C(GlobalPhase)",
+    "C(Hadamard)",
+    "C(IsingXX)",
+    "C(IsingXY)",
+    "C(IsingYY)",
+    "C(ISWAP)",
+    "C(MultiRZ)",
+    "ControlledQubitUnitary",
+    "C(PauliX)",
+    "C(PauliY)",
+    "C(PauliZ)",
+    "C(PhaseShift)",
+    "C(PSWAP)",
+    "C(Rot)",
+    "C(RX)",
+    "C(RY)",
+    "C(RZ)",
+    "C(S)",
+    "C(SWAP)",
+    "C(T)",
```

Comment:
These are not runtime operations (there is no gate named "C(Hadamard)" in the runtime interface), and I definitely don't want us embracing this format rather than moving away from it.

Reply:
In this PR I just replaced the runtime calculation of the same set with a hardcoded names. This allows me to remove [this call](https://github.com/PennyLaneAI/catalyst/pull/600/files#diff-49c4bb20bf2dcbd5fe316c548577fc168c4da1b075304b5bb953afcf8c4c7c25L77) which did the same.
I don't argue to keep these in the long term. As I see it, we remove the whole list when the new-style decomposition is ready

Reply:
@dime10 but I can remove this change if you really don't want to touch this. Let me know.